test/test.py

```diff
@@ -18,6 +19,13 @@
 
 TITLES_CSV = os.path.abspath(os.path.dirname(__file__)) + '/titles.csv'
 
+def waitForIndex(env, idx):
```

Comment:
I think we should allow some timeout here (default no timeout) and raise an exception if the timeout reached

Reply:
@MeirShpilraien 
Same script as the module.
We would remove once we index while loading rdb.

Reply:
I can see it will be useful in the future regardless of the tests (to create an index and wait for the reindexing or to alter and existing index and wait for the reindexing)